release/requirements.txt

```diff
@@ -1,5 +1,5 @@
 # ray
-https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-2.0.0.dev0-cp37-cp37m-manylinux2014_x86_64.whl
+https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-1.12.0rc0-cp37-cp37m-manylinux2014_x86_64.whl
```

Comment:
This shouldn't be changed I think. @krfricke to confirm.

Reply:
Confirmed this shouldn't be changed